src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -197,19 +206,29 @@ public Optional<Address> getAddress() {
             return Optional.ofNullable(address);
         }
 
+        public void setRemark(Remark remark) {
+            this.remark = remark;
+        }
+
+        public Optional<Remark> getRemark() {
+            return Optional.ofNullable(remark);
+        }
+
+        public void setBirthday(Birthday birthday) {
+            this.birthday = birthday;
+        }
+
+        public Optional<Birthday> getBirthday() {
+            return Optional.ofNullable(birthday).filter(date -> !date.getValue().equals(LocalDate.MIN));
```

Comment:
this check is redundant, itd be better to let optional handles this internally

Reply:
Fixed